pkg/virt-controller/watch/vmi_test.go

```diff
@@ -3294,7 +3299,7 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 
 			It("pod multus status cannot be updated", func() {
 				Expect(controller.updateMultusAnnotation(
-					vmi.Namespace, vmi.Spec.Domain.Devices.Interfaces, vmi.Spec.Networks, pod)).To(HaveOccurred())
+					vmi.Namespace, vmi.Spec.Domain.Devices.Interfaces[:1], vmi.Spec.Networks[:1], pod)).To(HaveOccurred())
```

Comment:
Why us this change required? The VMI has only one network/interface anyway.

Reply:
So why not changing the VM to have only one interface/network? It is the only test in this context.

Reply:
The point of the test is to change the multus annotation, from 2 interface to 1. And it should fail because the K8S API is down. It was not me that invented this test :)